k8s/openebs-operator.yaml

```diff
@@ -332,7 +332,7 @@ data:
         name: path filter
         state: true
         include: ""
-        exclude: "loop,/dev/fd0,/dev/sr0,/dev/ram,/dev/dm-"
+        exclude: "loop,/dev/fd0,/dev/sr0,/dev/ram,/dev/dm-,/dev/md-"
```

Comment:
Do we need /dev/loop  instead of loop ?

Reply:
it matches for substring. so loop should work. 

@umamukkara the suffix of "-" in the "/dev/md-" should be removed. 
(In case of device mapper, the paths will be /dev/dm-0 etc., But for md, looks like it is /dev/md0)